### PR TITLE
40/app

### PR DESCRIPTION
**Pull request by ChrisForti**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+then 
+ echo "npm is installed"
+else
+ echo "installing npm"
+ brew upgrade node
+fi
+
+# npm install
 
-if (npm version)
```
Comment (ChrisForti) [inline deploy.sh:5]: -version is not verbose, if we are only checking for desired state then verbose is not needed?

File: deploy.sh [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
 # Npm should be installed
+# gets installed with brew install node
+
+if (npm -version)
+then 
+ echo "npm is installed"
+else
+ echo "installing npm"
+ brew upgrade node
```
Comment (stephenlauck) [inline deploy.sh]: You have to upgrade node to get npm? Is npm already installed with node?

**Review by stephenlauck — CHANGES_REQUESTED** [review]
That's good you have the mac deploy done, but you should also have a linux (probably ubuntu) deploy script too for the environments we're going to build moving away from your local mac.

File: deploy.sh (hunk 2 above)
Comment (ChrisForti) [reply deploy.sh]: Yes it gets installed when you install node. So this code block is not even needed, I believe? I wrote to get your thoughts? When you upgrade node, it also upgrades npm.

Comment (stephenlauck) [reply deploy.sh]: Yeah just remove this block and then hit resolve on this issue.

Comment (ChrisForti) [reply deploy.sh]: That's a bet, I'll do it when I get by my machine.

**Pull request merged**